src/templates/Navigator/index.js

```diff
@@ -5,41 +5,11 @@ import classnames from 'classnames';
 import Layout from 'atoms/Layout';
 import Col from 'atoms/Layout/Col';
 import StepProgress from 'molecules/StepProgress';
+import { Partial, separatePartials, partialRenderer } from 'atoms/Partial';
```

Comment:
Moved to `/helper`. Agree the interface could be tighter, made a chore to do that along with unifying/ writing more tests.